packages/tools/fuzzer/model/Serializable.ts

```
@@ -0,0 +1,131 @@
+import { hasOwnProperty } from '@joplin/utils/object';
```

Biome (2.3.14)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)